contracts/TokenConversionLib.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity 0.4.23;
+
+import "./SafeMath.sol";
+
+library TokenConversionLib {
```

Comment:
We do not typically name our libraries, `...Lib`, so I suggest striking that from the name of this library.

And I suggest raising it as a question for Pro/the style guide, since we do name things `...Interface` and `...Abstract`.

Reply:
Please remove the `Lib` postfix.